hmda/Jenkinsfile

```diff
@@ -27,7 +28,7 @@ volumes: [
               sh "docker build --rm -t=${env.DOCKER_HUB_USER}/hmda-platform hmda/target/docker/stage"
               if (env.TAG_NAME != null || gitBranch == "master") {
                 if (gitBranch == "master") {
-                  env.DOCKER_TAG = "latest"
+                  env.DOCKER_TAG = "latest"+commitId
```

Comment:
We can give it a try, though I think the tag names need to be updated for helm to pick up changes. Whether we do `latest+SHA` or start versioning. 

I've updated the `--set image.tag=${env.DOCKER_TAG}` in the helm command incase we want to proceed with this change.

Reply:
I think the push to Docker Hub should probably just stay as `"latest"`, just to follow convention. But the one to DTR could use the `"latest"+commitId`, if that will trigger the deployment.

In that case we might want to variables, `DOCKER_HUB_TAG` and `DOCKER_REGISTRY_TAG`.

Reply:
The latest commits propose a much cleaner solution I found here: https://github.com/helm/helm/blob/master/docs/charts_tips_and_tricks.md#automatically-roll-deployments-when-configmaps-or-secrets-change  It would keep the docker tags the same but would restart based on sha256 id. Its worth a try.